contracts/yToken.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
+
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+
+import {VaultAPI, BaseWrapper} from "./BaseWrapper.sol";
+
+contract yToken is IERC20, BaseWrapper {
```

Comment:
Make a version that lets you do ETH <> WETH

Reply:
we can do it in a zap also

Reply:
Solved in d1ba7b0